cilium.yaml

```diff
@@ -0,0 +1,97 @@
+package:
+  name: cilium
+  # When updating this version, please check the dependencies patches below
+  # and remove if possible.
+  version: 1.14.2
+  epoch: 0
+  description: Cilium is a networking, observability, and security solution with an eBPF-based dataplane
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - iptables
+      - iproute2
+      - ipset
+      - bpftool
+      - kmod
+      # cilium does compilations at runtime on the node.
+      - clang
+      - llvm16
+      - cni-plugins
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - build-base
+      - go
+      - git
+      - coreutils # for GNU install
+      - grep
+      - clang
+      - llvm16
+      - iptables # for cilium-iptables
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/cilium/cilium
+      tag: v${{package.version}}
+      expected-commit: a67489466c715854c4c472c939d32d35df3fe748
+
+  - runs: |
+      # For CVE-2023-39325 ,CVE-2023-3978 and CVE-2023-44487
+      go get golang.org/x/net@v0.17.0
+      go mod tidy
+      go mod vendor
+
+      # Remove groupadd from Makefile: it's not doing anything useful in
+      # a package build anyway, and it's not available in busybox.
+      find . -name Makefile -exec sed -i '/groupadd/d' {} \;
+
+      DESTDIR=${{targets.destdir}} DISABLE_ENVOY_INSTALLATION=1 make build-container
+      DESTDIR=${{targets.destdir}} DISABLE_ENVOY_INSTALLATION=1 make install-container
+
+  - uses: strip
+
+subpackages:
+  - name: cilium-iptables
```

Comment:
nit, should this be, and below:
${{package.name}}-iptables
${{package.name}}-operator-generic?

Reply:
Great catch! Fixing!

Reply:
Done!